### ISSUE TITLE
Text in labels not shown correctly

### ISSUE THREAD

**Issue by Arc676** [Arc676/xkcd-Viewer#8]
The JSON data includes the alt text in plaintext, but includes escape sequences for special characters or lacks escape characters for other characters.

This results in accented characters not being displayed correctly. Additionally, HTML tags in the title text (comic 1144) aren't shown.

**Comment by Arc676** [comment]
This has become an issue again in the new release. See comic 259.

**Comment by Arc676** [comment]
`&nbsp;` (1144) and `&quot;` (2332) don't get parsed properly.

**Comment by Arc676** [comment]
Marking 1144 as intended behavior since `&nbsp;` also appears when viewing the comic in Firefox.

**Comment by Arc676** [comment]
Greater-than signs don't render properly in comic 2143.

**Comment by Arc676** [comment]
The `&amp;` in 2437 doesn't render properly.
